frontend_tests/node_tests/composebox_typeahead.js

```diff
@@ -480,9 +490,14 @@ test("content_typeahead_selected", ({override_rewire}) => {
 
     fake_this.query = "@_back";
     fake_this.token = "back";
-    with_field(compose_validate, "warn_if_mentioning_unsubscribed_user", unexpected_warn, () => {
-        actual_value = ct.content_typeahead_selected.call(fake_this, backend);
-    });
+    with_field_rewire(
```

Comment:
I think every time that we map `with_field_rewire` to a function, we are just doing a weaker version of `override_rewire`, so those might be some early targets for a code audit.  (Even if all we do in the short term is cut them over to `override_rewire`, we can at least remove some arbitrariness.)